aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>You've got some hard-coded stuff here to discuss.</p><p>Tell the customer what instance type and instance count minimum and recommendation. You mention type, but this code doesn't specify count, unless I'm supposed to know that 32 is count. You can pass the variable names in for both, so do so, so it is clear what these parameters mean.</p><p> </p><p>The transform call is using some s3 bucket that I don't remember setting up... I see /pump but not /pump/pump</p><p> </p><p>I'm a little concerned about this Transform procedure being difficult to follow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>